collab-document/src/blocks/children.rs

```diff
@@ -0,0 +1,98 @@
+use collab::preclude::*;
+use serde::ser::SerializeMap;
+use serde::{Serialize, Serializer};
+
+pub struct ChildrenMap {
+  pub root: MapRefWrapper,
+}
+
+impl Serialize for ChildrenMap {
+  fn serialize<S>(&self, serializer: S) -> Result<S::Ok, S::Error>
+  where
+    S: Serializer,
+  {
+    let txn = self.root.transact();
+    let mut map = serializer.serialize_map(Some(self.root.len(&txn) as usize))?;
+    for (key, _) in self.root.iter(&txn) {
+      let children = self.root.get_array_ref_with_txn(&txn, key).unwrap();
```

Comment:
If you sure this is ok to unwrap, you can comment on top of the unwrap line. for example:

// It's safe to unwrap, because xxxx